src/com/rollbar/notifier/RollbarNotifier.as

```diff
@@ -131,8 +131,8 @@ package com.rollbar.notifier {
             return handleError(newError);
         }
 
-        public function handleOtherEvent(event:*):String {
-            var newError:Error = new Error("A non-Error or ErrorEvent was thrown and not caught: " + event.toString());
+        public function handleOtherEvent(error:*):String {
+            var newError:Error = new Error("A non-Error or ErrorEvent was thrown and not caught: " + error.toString());
```

Comment:
@pasviegas Would it be more useful to send both the event and the error? e.g. include both `event.toString()` and `event.error.toString()`